app/main.py

```diff
@@ -1 +1,58 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
Defining `skills` as a class variable here can lead to shared state across all instances of `SoftwareEngineer`. Consider removing this line to avoid potential issues with shared state.